tools/k4aviewer/k4apointcloudwindow.cpp

```diff
@@ -92,7 +92,7 @@ void K4APointCloudWindow::Show(K4AWindowPlacementInfo placementInfo)
     }
 
     ImGui::SameLine();
-    ImGui::VerticalSeparator();
+    ImGui::SeparatorEx(ImGuiSeparatorFlags_Vertical);
```

Comment:
I don't object to this, but I'd like to understand why we're making the change so I can do the right thing in future - what's the deal here?

Reply:
`VerticalSeparator()` was from imgui_internal.h and was changed in the latest release. This is the equivalent behavior in the latest version.